src/vivarium_gates_nutrition_optimization/constants/metadata.py

```diff
@@ -15,7 +15,7 @@
 MAKE_ARTIFACT_RUNTIME = "3:00:00"
 MAKE_ARTIFACT_SLEEP = 10
 
-LOCATIONS = ["Ethiopia", "Nigeria", "Pakistan"]
+LOCATIONS = ["Ethiopia"] #, "Nigeria", "Pakistan"]
```

Comment:
Do we want this to be included in this PR? Or should we just modify when we are running locally? Not sure what is best practice.

Reply:
Good question - would love engineering input on best practice for things like this.

Reply:
I think it's fine to include all 3 locations. Doing that will give you the option to run all 3 locations but which location you actually build is only specified when you actually run make_artifacts, so including all of them here won't hurt anything.

Reply:
Thanks! I'll change this to include all 3 locations.